src/components/app/Home.js

```diff
@@ -502,6 +481,28 @@ class Home extends React.PureComponent<HomeProps, HomeState> {
           <TransitionGroup className="homeInstructionsTransitionGroup">
             {this._renderInstructions()}
           </TransitionGroup>
+          <section className="homeAdditionalContent">
+            <h2 className="homeAdditionalContentTitle photon-title-30">
```

Comment:
Please add comments here as to the layout intent.

`{/* Full width */}`

...

`{/* Grid Column 1 */}`

...

`{/* Grid Column 2 */}`